### ISSUE TITLE
Inquiry regarding the training data from EmoBox used to fine-tune emotion2vec+ seed.

### ISSUE THREAD

**Issue by jeeyung** [ddlBoJack/emotion2vec#77]
Hello!
Thank you for sharing the implementations.
I was wondering if you could provide the training data (from EmoBox) used for emotino2vec+ seed. I want to test the model on EmoBox and would like to use data that was not used for fine-tuning.
